CHANGELOG.md

```diff
@@ -7,18 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.1.10] 2024-08-01
+
+### Added
+
+- Added `/search/index` and `/collections/slug` pages (DR-3028)
+- Added `/items` pages (DR-3029)
+
+### Updated
+
+- Updated static breadcrumbs to render correct copy (DR-3023)
+- Updated `/divisions` page to render title and content (DR-3030)
+
 ## [0.1.9] 2024-07-08
 
 ### Updated
 
+- Updates card columns to new breakpoints (DR-3036)
```

Comment:
Were these missed in the previous deployment or should they move up?

Reply:
they were missed in the changelog

Reply:
in the previous release

Reply:
or maybe I'm wrong. maybe it was just the tooltip or making the card clickable. lemme double check

Reply:
updated